hvplot/plotting/scatter_matrix.py

```diff
@@ -38,19 +46,86 @@ def scatter_matrix(data, c=None, chart='scatter', diagonal='hist', alpha=0.5, **
     diagonal = HoloViewsConverter._kind_mapping[diagonal]
     chart = HoloViewsConverter._kind_mapping[chart]
 
+    #remove datashade kwds
+    if datashade or rasterize:
+        ds_kwds = {}
+        if 'aggregator' in kwds:
+            ds_kwds['aggregator'] = kwds.pop('aggregator')
+        
+    #remove dynspread kwds
+    if dynspread:
+        if hd.ds_version < '0.11.1':
+            print('WARNING: Any version of datashader less than 0.11.1 will not support dynspread.')
```

Comment:
@jlstevens I added in some warnings about the datashader version, but I am unsure of the version exactly.  I tried to make my best guess based on the repo, but I haven't installed older versions to test it.  Also, should this be an error?  Or should I be using a logger?

Reply:
I'll get back to you on the exact version. As for warning vs exception, I think asking for `spread` or `dynspread` in the `rasterize` case is an error for those old versions as the user requested something that the function cannot deliver. I would raise an appropriate Exception (I haven't thought of what exactly) that includes the message you wrote above but also explains that the issue isn't asking for `dynspread` but asking for `dynspread` *with* `rasterize`.

Reply:
I just confirmed that 0.11.1 also doesn't have the necessary changes 0.12.0 should though. So I would use `< '0.12.0` here. Also, is `hd.ds_version` also a string? If so, I would use one of the version comparison utilities as string version comparison is unsafe.

Reply:
hd.ds_version is a LooseVersion.  I was just looking up some different ways to do this, and will make a change.

Reply:
Ok great! Was just checking...

Reply:
@jlstevens I looked around the repo and it seems that for all the looseVersions of various packages, its compared to a string.  Is there a different way I should do the version comparison?

Reply:
No, you're good; Jean-Luc was just checking that ds_version is already a LooseVersion, because if it were not, there would be problems.

Reply:
Great, thanks @jbednar !